### PR TITLE
[Fix #3114] Fix alignment of end in EmptyElse

### LINKED ISSUE #3114: Autocorrect align end to if is misaligned
running rubocop on this file with auto-correct

```
a = if x == 1
      "A"
    elsif x == 2
      "B"
    else
      nil
    end
```

results in 

```
a = if x == 1
      "A"
    elsif x == 2
      "B"
        end
```

Running it again will not align the end

```
test.rb:5:9: W: end at 5, 8 is not aligned with if at 1, 4.
        end
        ^^^
```

### PR DESCRIPTION
**Pull request by rrosenblum**
This fixes #3114. I also found some redundancies in the `both` check.

### CODE REVIEW

**Comment by bbatsov** [comment]
alignment

**Comment by rrosenblum** [comment]
The spelling mistake has been fixed.

**Comment by tejasbubane** [comment]
The PR subject as well

**Comment by rrosenblum** [comment]
I have fixed the subject as well

File: CHANGELOG.md [hunk 1]
```diff
@@ -52,6 +52,7 @@
 * [#3063](https://github.com/bbatsov/rubocop/issues/3063): Don't auto-correct `a + \` into `a + \\` in `Style/LineEndConcatenation`. ([@jonas054][])
 * [#3034](https://github.com/bbatsov/rubocop/issues/3034): Report offenses for `RuntimeError.new(msg)` in `Style/RedundantException`. ([@jonas054][])
 * [#3016](https://github.com/bbatsov/rubocop/issues/3016): `Style/SpaceAfterComma` now uses `Style/SpaceInsideHashLiteralBraces`'s setting. ([@ptarjan][])
+* [#3114](https://github.com/bbatsov/rubocop/issues/3114): Fix alignment `end` when auto-correcting `Style/EmptyElse`. ([@rrosenblum][])
```
Comment (bbatsov) [inline CHANGELOG.md]: We've cut a new release.

**Comment by rrosenblum** [comment]
I have moved the `CHANGELOG` entry to master.

**Pull request merged**